client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMeta.vue

```diff
@@ -232,8 +232,8 @@ export default {
       menuEntries: [
         { id: 'entry', transKey: 'entry' },
         { id: 'submitter', transKey: 'submitted.author' },
-        { id: 'publicationAndVoting', transKey: 'publication.and.voting' },
-        { id: 'locationAndDocuments', transKey: 'location.and.document.reference' },
+        { id: 'publicationAndVoting', transKey: 'publication.and.voting', condition: hasPermission('feature_statements_vote') || hasPermission('feature_statements_publication') },
```

Comment:
```suggestion
        { id: 'publicationAndVoting', transKey: 'publication.and.voting', condition: hasAnyPermissions(['feature_statements_vote', 'feature_statements_publication']) },
```
but then You have to add `hasAnyPermissions` in the import section in he demosplan-ui block

Reply:
done in https://github.com/demos-europe/demosplan-core/pull/4130/commits/cad05829d871fdd55860c7a93731f3bc7bacfa58

Reply:
sorry, but you did not read the comment below my suggestion

Reply:
Sorry, done in https://github.com/demos-europe/demosplan-core/pull/4130/commits/4098c6c297bf40d01f3c0612895ba93b8d103a41